test/RibbonThetaVault.js

```diff
@@ -2270,12 +2304,10 @@ function behavesLikeRibbonOptionsVault(params) {
 
         const balanceBeforeWithdraw = await this.assetContract.balanceOf(user);
 
-        const [
-          withdrawAmount,
-          feeAmount,
-        ] = await this.vault.withdrawAmountWithShares(
-          BigNumber.from("10000000000")
-        );
+        const [withdrawAmount, feeAmount] =
+          await this.vault.withdrawAmountWithShares(
+            BigNumber.from("10000000000")
+          );
 
         assert.equal(withdrawAmount.toString(), BigNumber.from("9950000000"));
         assert.equal(feeAmount.toString(), BigNumber.from("50000000"));
```

Comment:
*tests wont work yet - opyn team waiting on us to whitelist yvweth as collateral

Reply:
Ya let's get it whitelisted for collateral. To make the tests work however, we can [impersonate the whitelist's owner](https://github.com/ribbon-finance/structured-products/blob/master/test/helpers/utils.js#L237) then do `whitelistCollateral` and `whitelistProduct` ourselves.